android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -202,28 +203,66 @@ public Object getSelectionKey()
 				}
 			},
 			StorageStrategy.createLongStorage()
-		)
-			.withSelectionPredicate(SelectionPredicates.<Long>createSelectSingleAnything())
-			.build();
-		this.tracker.addObserver(new SelectionTracker.SelectionObserver()
-		{
```

Comment:
When you set "editing" to `true` but none of the rows are checked, then the end-user has to long-press a row to check a row's checkbox. Once at least 1 row is checked, the user is then able to tap each row to check them. This is a confusing interface. Especially since unchecking all rows means you have to long-press a row to enable checking them again.

There is an easy solution to this. Add the following method to where we create the `ItemDetails` class up above.
```java
return new ItemDetails() {
	// ... your existing methods ...

	@Override
	public boolean inSelectionHotspot(@NonNull MotionEvent e)
	{
		// Returning true allows taps to immediately select this row.
		return true;
	}
};
```